p2p/ipld/read.go

```diff
@@ -27,7 +25,7 @@ var ErrTimeout = fmt.Errorf("%s %s", baseErrorMsg, "timeout")
 func RetrieveBlockData(
 	ctx context.Context,
 	dah *types.DataAvailabilityHeader,
-	api coreiface.CoreAPI,
+	dag format.NodeGetter,
```

Comment:
```suggestion
	nodeGetter format.NodeGetter,
```
?

Reply:
Naaah, too long. I don't understand why it is more "descriptive" to repeat words:
>  nodeGetter format.NodeGetter

Here we use words "node" and "getter" two times and I try to avoid that

Reply:
OK